docs/changelog.md

```diff
@@ -1,6 +1,14 @@
 # Changelog
 
-## *v0.20* (2019-04-17):
+## [0.21.0] -- (2019-05-02)
+
+### Added
+- Support for Snakemake projects
+
```

Comment:
Is there anything else we need to add to changelog @vreuter for this release?

Reply:
Hmm let me double-check